src/kOS/Function/Misc.cs

```diff
@@ -28,6 +28,51 @@ public override void Execute(SharedObjects shared)
         }
     }
 
+[Function("hudtxt")]
+    public class FunctionHudTxt : FunctionBase
+    {
+
+        public override void Execute(SharedObjects shared)
+
+        {
+            int mirror          = Convert.ToInt32(shared.Cpu.PopValue());
+            string color        = shared.Cpu.PopValue().ToString();
```

Comment:
To make this work (since you're using HTML color tags to communicate this), I could provide you with a ToHTMLColor() method from RgbaColor that, for example, if you have RGB(1.0,0.5,0.5) would out put a string like "#ff8080".